pkg/proxy/endpoints.go

```diff
@@ -419,7 +419,7 @@ func (ect *EndpointChangeTracker) endpointsToEndpointsMap(endpoints *v1.Endpoint
 		for i := range ss.Ports {
 			port := &ss.Ports[i]
 			if port.Port == 0 {
-				klog.Warningf("ignoring invalid endpoint port %s", port.Name)
+				klog.InfoS("Ignoring invalid endpoint port", "portName", port.Name)
```

Comment:
I'd also say Error here - this should not happen!

Reply:
Done! Thanks :)